cmd/prometheus-config-reloader/main.go

```diff
@@ -122,25 +121,31 @@ func main() {
 		os.Exit(0)
 	}
 
-	logger, err := logging.NewLogger(logConfig)
+	logger, err := logging.NewLoggerSlog(logConfig)
+	if err != nil {
+		stdlog.Fatal(err)
+	}
+
+	// The old logger will be removed in the future. For now, we are using it to avoid breaking changes and
+	// to have a smooth transition.
+	oldLogger, err := logging.NewLogger(logConfig)
```

Comment:
```suggestion
	// We're currently migrating our logging library from go-kit to slog.
	// The go-kit logger is being removed in small PRs. For now, we are creating 2 loggers to avoid breaking changes and
	// to have a smooth transition.
	goKitLogger, err := logging.NewLogger(logConfig)
```

Reply:
Could you make similar changes to the other two `main.go` files?

Reply:
For sure!